CMakeLists.txt

```diff
@@ -210,6 +217,7 @@ if(NOT WIN32)
         core_optional_dep(LibDRM ENABLE_MIR)
       else()
         core_optional_dep(X ENABLE_X11)
+        core_optional_dep(XRandR ENABLE_X11)
```

Comment:
@mac-l1 thanks much, things are getting clearer now.

1. Could you please separate X11 related changes into a new PR? I tihnk those are ok and can go in.

2. Are you saying there is nother AML specific in EGLNativeTypeAmlogic? If this is the case we maybe should rename it to something more generic.